tests/switchmap_/server/db/misc/test_interface.py

```
@@ -0,0 +1,190 @@
+    sys.path.insert(0, ROOT_DIR)
+else:
+    print(
+        """This script is not installed in the "{0}" directory. Please fix.\
+""".format(
+            _EXPECTED
+        )
+    )
+    sys.exit(2)
+
+# Import test dependencies
+from tests.testlib_ import setup
```

Ruff (0.8.2)

`tests.testlib_.setup` imported but unused

Remove unused import: `tests.testlib_.setup`

(F401)